projects/max14906/src/platform/maxim/parameters.h

```diff
@@ -80,6 +81,7 @@ extern struct max_spi_init_param max14906_spi_extra;
 #define GPIO_OPS	&max_gpio_ops
 #define GPIO_EXTRA	&max14906_gpio_extra_ip
 #define GPIO_IRQ_OPS	&max_gpio_irq_ops
+#define GPIO_IRQ_EXTRA	NULL
```

Comment:
ditto.

Reply:
GPIO_IRQ_EXTRA in case of maxim should be null.